.ci_support/win_64_.yaml

```diff
@@ -16,8 +16,6 @@ cxx_compiler:
 - vs2019
 target_platform:
 - win-64
-vc:
-- '14'
```

Comment:
Oddly re-rendering is removing this

Think this may be leading to issues

Reply:
Worked around with an older conda-smithy and friends

We should investigate later. Filed issue: https://github.com/conda-forge/cuda-nvcc-impl-feedstock/issues/32